### DISCUSSION TITLE
d3d_estuary_ENKF

### DISCUSSION THREAD

**Discussion by uv-moscow** [OpenDA-Association/OpenDA#307] [Q&A]
Hello,

I try to execute example “d3d_estuary_ENKF” and get exception “Java.lang.RuntimeException: NetcdfD3dMapDataObject: Sigma layers detected, only z-layers currently supported for Delft3d-Flow”.

Does this mean that the existing solution only works with the z-model, or do I need to make some changes to work with the sigma-model?

**Comment by erikpelgrim** [comment]
Hi,

I had a quick look at this and other delft3d examples and it seems like some need updating.
Unfortunately we currently do not have the capacity to do this very soon.
Apologies for the inconvenience.

**Reply by uv-moscow** [reply]
Thank you for reply!

**Reply by erikpelgrim** [reply]
Hi,

I checked version 2.4.5 of OpenDA which can be downloaded from https://sourceforge.net/projects/openda/files/
And there the d3d_estuary_EnKF example does seem to work, maybe that is helpful.
